src/redux/sagas/setupUserContext.ts

```diff
@@ -61,6 +62,19 @@ export default function* setupUserContext() {
       type: ActionTypes.WALLET_OPEN_SUCCESS,
     });
 
+    /*
+     * Get saved decision from local storage and, if it exists, add to the store.
+     */
+
+    const decision = getDecisionFromLocalStorage(wallet?.address || '');
```

Comment:
Whenever a new wallet is opened, the decision held in local storage will be saved to the store.